cs/test/core/ConvertTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿namespace UnitTest.Convert
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+    using System.Threading;
+    using Bond;
+    using NUnit.Framework;
```

Comment:
Minor: put the Bond usings near each other.

Reply:
Done.